run-all-available-rules.md

```diff
@@ -0,0 +1,51 @@
+---
+description: How do I run all available rules on my repository?
+tags:
+  - Rules
+  - Semgrep Registry
+  - Semgrep Code
+---
+
+# How to run all available rules on a repository.
+
+If you would like to run all rules that are available within the [Semgrep Registry](https://semgrep.dev/explore) and are not willing to add all available rules/rulesets to the Rule Board or add a variety of `--config` entries, what you can do is run the following command:
+
+```
+semgrep --config=r/all
+```
+
+This command will pull out all avaialable public, private, and pro tiered rules if you have access to them. Below is an example output:
```

Comment:
```suggestion
This command will pull out all available public, private, and pro tiered rules if you have access to them. Below is an example output:
```
Curious about the scope of this command: Is the goal of r/all to also run private rules from _local_ sources, such as .semgrep?

Thank you Jason.